adobe/spectrum-web-components
# docs: Add custom vars table to documentation site

## Description

Created a custom variable table that allows users to view colours. This lives as its own app within the projects folder, but is also live on the documentation for `sp-theme`. I chose to add it to the Theme documentation since we link there directly from the "Getting started" page, and I felt that adding a new page would lead to less visibility. 

## Related issue(s)

fixes https://github.com/adobe/spectrum-web-components/issues/1458

## Motivation and context


Improving onboarding experience for new users. Improve understanding of how sp-theme works. Documenting custom properties somewhere more accessible. 
## How has this been tested?




- Go to documentation for sp-theme
- Scroll down to "What it's doing"
- See that the table loads. 
- Change theme via picker and see that colour values change.
- Scroll back up, change the documentation site's theme
- See that the theme of the colour values also changes (unless you have wireframe selected)

## Screenshots (if appropriate)

## Types of changes




## Checklist




## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.
